build/_build.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net5.0</TargetFramework>
+        <RootNamespace></RootNamespace>
+        <NoWarn>CS0649;CS0169</NoWarn>
+        <NukeRootDirectory>..</NukeRootDirectory>
+        <NukeScriptDirectory>..</NukeScriptDirectory>
+        <NukeTelemetryVersion>1</NukeTelemetryVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Nuke.Common" Version="5.2.1"/>
+        <PackageDownload Include="GitVersion.Tool" Version="[5.6.7]"/>
+        <PackageReference Include="Nuke.OctoVersion" Version="0.2.560"/>
+        <PackageReference Include="OctoVersion.Core" Version="0.2.560"/>
+        <PackageDownload Include="NuGet.CommandLine" Version="[5.9.1]"/>
+    </ItemGroup>
```

Comment:
nitpick: ordering can go a long way to being able to read and reason about references.

I suspect there are also more references than required here. Shouldn't `Nuke.OctoVersion` bring in `OctoVersion.Core` for you? IS the `GitVersion.Tool` reference just there to get around the net5.0 issues with OctoVersion?
We've moved away from OctoVersion for other Open Source projects, and found things much simpler. E.g. https://github.com/OctopusDeploy/AzureDevOpsIssueTracker/blob/main/build/_build.csproj

Reply:
Ah yeah - thanks for catching that. I think those came in whilst getting my env sorted. 🙏 I can remove both `GitVersion.Tool` and `OctoVersion.Core`.